javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -42,7 +43,7 @@ public void testAsserts() {
 
     public static void main(String... args) {
         JmhRunner.runDebugWithAsserts(CLASSES);
-        JmhRunner.runNormalNoAsserts(CLASSES);
+        JmhRunner.runNormalNoAsserts(CLASSES, JAVA, FUNCTIONAL_JAVA, PCOLLECTIONS, CLOJURE, SCALA, JAVASLANG);
```

Comment:
During optimizations I don't want to wait for `pcollections` and `fjava`, so I just delete these temporarily:
```java
JmhRunner.runNormalNoAsserts(CLASSES, SCALA, JAVASLANG);
```